keyboards/eason/void65h/readme.md

```diff
@@ -0,0 +1,24 @@
+# void65h
+
+![void65h](https://i.imgur.com/3Z3jNAS.jpg)
+
+A customizable hotswap 65% keyboard.
+
+* Keyboard Maintainer: [EASON](https://github.com/EasonQian1)
+* Hardware Supported: void65h
+* Hardware Availability: [vertex-kb](https://github.com/Vertex-kb)
```

Comment:
Is there an actual public repo for this, or otherwise a link for a place to purchase?

Reply:
Corrected